measures/BuildExistingModel/measure.rb

```diff
@@ -210,9 +234,7 @@ def run(model, runner, user_arguments)
       end
     end
 
-    # Get the absolute paths relative to this meta measure in the run directory
-    new_runner = OpenStudio::Measure::OSRunner.new(OpenStudio::WorkflowJSON.new) # we want only ResStockArguments registered argument values
-    if not apply_measures(measures_dir, { 'ResStockArguments' => measures['ResStockArguments'] }, new_runner, model, true, 'OpenStudio::Measure::ModelMeasure', nil)
+    def register_logs(runner, new_runner)
```

Comment:
Good idea wrapping this in a method. But I'd probably have this method outside the measure's `run` method.